### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jozefv-git**
Name added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# TatraTeam
\ No newline at end of file
+# TatraTeam
+## Jozef Visnovec
```
Comment (CezarBudeci) [inline README.md:2]: Make it a list item

**Review by Smoothie16 — APPROVED** [review]
Looks good to me

File: README.md (hunk 1 above)
Comment (CezarBudeci) [inline README.md:2]: Make it a list item

**Review by CezarBudeci — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jozefv-git) [reply README.md:2]: How?

File: index.html [hunk 2]
```diff
@@ -6,5 +6,6 @@
   </head>
   <body>
     <p><b>TatraTeam</b></p>
+    <li>Jozef Visnovec</li>
```
[the remark this answers is not in the document]
Comment (CezarBudeci) [reply index.html:9]: naah we not having list

**Review by CezarBudeci — CHANGES_REQUESTED** [review]
take out the list

**Review by CezarBudeci — APPROVED** [review]
LGTM!

**Comment by jozefv-git** [comment]
Resolved confligts

**Pull request merged**